shells/dev-shell/index.js

```diff
@@ -73,9 +73,12 @@ async function wrappedExecute() {
     arcPanel.attachArc(arc);
 
     const resolver = new RecipeResolver(arc);
-    const resolvedRecipe = await resolver.resolve(recipe);
+    const options = {errors: new Map()};
+    const resolvedRecipe = await resolver.resolve(recipe, options);
```

Comment:
`resolve()` doesn't take an options object.

Reply:
I believe the follow is the current implementation (which does take a options value.

src/runtime/recipe/recipe-resolver.ts:179
```
  // Attempts to run basic resolution on the given recipe. Returns a new
  // instance of the recipe normalized and resolved if possible. Returns null if
  // normalization or attempting to resolve slot connection fails.
  async resolve(recipe: Recipe, options?: IsValidOptions ) {
```

Reply:
Ah, managed to split my commits incorrectly. Uploading the rest of the change.